docs/policy/software-support.md

```diff
@@ -39,7 +39,7 @@ Note that if you follow the above, we will end up with three assignees to each t
     2.  Software Support (Triage)
     3.  The Software team member who will handle the ticket
 
-    Inasmuch as possible, you should strive to handle the easier tickets and not pass them off to other people. For reference, see our [troubleshooting documents](https://twiki.grid.iu.edu/bin/view/Documentation/Release3/#Software_Guides_Troubleshooting).
+    Inasmuch as possible, you should strive to handle the easier tickets and not pass them off to other people. For reference, see our [troubleshooting documents](/docs).
```

Comment:
Since this is a cross-link to the docs area, you'll need to provide the full link: https://opensciencegrid.github.io/docs/